tests/test_sensors.py

```diff
@@ -34,8 +33,28 @@ def test_imu_sensor(show_viewer):
         ),
     )
 
-    imu_biased = scene.add_sensor(IMUOptions(entity_idx=box.idx, accelerometer_bias=BIAS, gyroscope_bias=BIAS))
-    imu_delayed = scene.add_sensor(IMUOptions(entity_idx=box.idx, read_delay=DT * 2))
+    imu_biased = scene.add_sensor(gs.sensors.IMU(entity_idx=box.idx, acc_bias=BIAS, gyro_bias=BIAS))
```

Comment:
I'm not convinced that stripping away to `Options` suffix is a good idea. Personally I find it confusing. But no strong opinion.